.github/workflows/release.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         id: changesets
         uses: FuelLabs/changesets-action@main
         with:
-          publish: pnpm changeset:release
+          publish: pnpm changeset publish --tag next
```

Comment:
On fuels-npm-packs we don't have pre-release next / rc. We should update the tag `main` to `next`. And keep this one as release latest.

Reply:
https://github.com/FuelLabs/fuels-npm-packs/pull/61/commits/ae973fe5ac3dc82c8190e6c30e6aca231765e021